Source/WebCore/PAL/pal/spi/cg/CoreGraphicsSPI.h

```diff
@@ -409,6 +409,7 @@ extern CGDataProviderRef CGDataProviderCreateMultiRangeDirectAccess(
 #if HAVE(LOCKDOWN_MODE_PDF_ADDITIONS)
 CG_EXTERN void CGEnterLockdownModeForPDF();
 CG_LOCAL bool CGIsInLockdownModeForPDF();
+CG_EXTERN void CGEnterLockdownModeForFonts();
```

Comment:
CC @brentfulgham. As far as I can tell, this needs to be added in addition to the Lockdown Mode SPI we landed in #2426. Perhaps it shouldn't be under `LOCKDOWN_MODE_PDF_ADDITIONS`, but we don't appear to have a fonts-specific PlatformHave macro.

Reply:
I don't really think SPI headers *need* #ifs.